src/app/page.tsx

```
@@ -293,58 +301,159 @@ export default function Home() {
             });
             loadContacts();
           }
+
+// Send money from RequestsTab form (with optional save as contact)
+const sendMoneyFromRequests = useCallback((toAddress: string, amount: number, memo?: string, saveAsContact?: boolean, contactLabel?: string) => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)